tor/main.py

```diff
@@ -48,7 +48,8 @@ def run(config):
 
 def main():
     config.debug_mode = bool(os.environ.get('DEBUG_MODE', False))
-    bot_name = 'debug' if config.debug_mode else os.environ.get('BOT_NAME', 'bot')
+    bot_name = 'debug' if config.debug_mode \
+        else os.environ.get('BOT_NAME', 'bot')
```

Comment:
Since it explodes out to multiple lines (due to line length), why don't we just convert this into a fully-fledged `if`-`else` conditional block here?